api-spec/STAC-standalone.yaml

```diff
@@ -0,0 +1,395 @@
+openapi: 3.0.1
+info:
+  title: STAC API without WFS3

```

Comment:
I'd prefer more just 'standalone stac api', so a dev totally new to this doesn't have to figure out what wfs3 is to get their head around it.